react/components/BaseModal.tsx

```diff
@@ -1,9 +1,10 @@
-import React, { useState } from 'react'
+import React, { useState, useRef } from 'react'
```

Comment:
I don't know if that is something too important, but if you try to close the first modal that you open in the page with the back button it will reload the page and scroll up, after that it works perfectly. I try to find out why, but I couldn't